src/java.base/share/classes/java/io/RandomAccessFile.java

```diff
@@ -758,9 +758,9 @@ public final boolean readBoolean() throws IOException {
      * If the byte read is {@code b}, where
      * {@code 0 <= b <= 255},
      * then the result is:
-     * <blockquote><pre>
-     *     (byte)(b)
-     * </pre></blockquote>
+     * {@snippet lang=java :
+     *         (byte)(b)
```

Comment:
4 space indent is sufficient in this file

Reply:
So changed in b6abb2e40fe574f40ebdfc0c5b1e28ed4b09126e.